policy-controller/k8s/index/src/lib.rs

```diff
@@ -63,6 +63,9 @@ pub struct ClusterInfo {
 
     /// The cluster-wide default protocol detection timeout.
     pub default_detect_timeout: time::Duration,
+
+    /// The networks that probes are expected to be from.
```

Comment:
if there is a difference in semantics between a `None` and an empty `Vec` here, it would be nice if the comment explained it.

Reply:
This field is no longer an `Option`.